packages/currency/src/currency-utils.ts

```diff
@@ -27,7 +27,7 @@ export const isValidNearAddress = (address: string, network?: string): boolean =
   switch (network) {
     case 'aurora':
     case 'near':
-      return !!address.match(/\.near$/);
+      return !!address.match(/\.(near|tg)$/);
```

Comment:
@yomarion looks like we did need a protocol change for: https://app.asana.com/0/1207673178190193/1208099463835050 - I have made changes here too: https://github.com/RequestNetwork/currency-api/pull/571/files

Can you think of anything else we need to change to support .tg?

Reply:
Maybe add a unit-test in the lib to be 100% sure this part is done, but yes it looks OK to me.

Reply:
@AdamREQ ☝️